examples/web/.env.example

```
@@ -1,2 +1 @@
-VITE_SERVER_URL=""
-OPEN_ROUTER_API_KEY=""
\ No newline at end of file
+VITE_SERVER_URL=""
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)